### PR TITLE
perf(dc): improve err msg when cannot find domain for auth API

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

**Review by KennethBWSong — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -901,6 +901,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Invalid client secret. It should be 10 to 512 characters long.",
   "driver.apiKey.error.domainInvalid": "Invalid domain. Please follow these rules: 1. Max %d domain(s) per API key. 2. Use comma to separate domains.",
   "driver.apiKey.error.failedToGetDomain": "Unable to get domain from API specification. Make sure your API specification is valid.",
+  "driver.apiKey.error.authMissingInSpec": "No API in the OpenAPI specification file contains API key authentication with the name %s.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:904]: Proposed content: No API in the OpenAPI specification file matches the API key name '%s'. Please verify the key name.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -913,7 +914,8 @@
   "driver.oauth.error.OauthIdentityProviderInvalid": "Invalid identity provider 'MicrosoftEntra'. Ensure the OAuth authorization endpoint in the OpenAPI specification file is for Microsoft Entra.",
   "driver.oauth.log.successCreateOauth": "OAuth registration created successfully with id %s!",
   "driver.oauth.error.domainInvalid": "Maximum %d domains allowed per OAuth registration.",
-  "driver.apiKey.error.oauthAuthInfoInvalid": "Unable to parse OAuth2 authScheme from spec. Make sure your API specification is valid.",
+  "driver.oauth.error.oauthAuthInfoInvalid": "Unable to parse OAuth2 authScheme from spec. Make sure your API specification is valid.",
+  "driver.oauth.error.oauthAuthMissingInSpec": "No API in the OpenAPI specification file contains OAuth authentication with the name %s.",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:918]: Proposed content: No API in the OpenAPI spec uses OAuth authentication named '%s'. Please verify the name in the specification.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json]: Thanks, updated

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json]: Thanks, updated

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**